.gitignore

```diff
@@ -6,7 +6,9 @@
 !/cmake/*.cmake
 *~
 /test/benchmark_test
+/test/benchmark_test.exe
```

Comment:
Why do we need these? I don't think we should be building in-tree like this.
Either way the stuff already seems to be there for Unix so I guess this is fine.